interfaces/IMessageControl.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2022 Metrological
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "Module.h"
+
+namespace WPEFramework {
+namespace Exchange {
+
+struct EXTERNAL IMessageControl : virtual public Core::IUnknown {
+  virtual ~IMessageControl() = default;
+  enum { ID = ID_MESSAGE_CONTROL };
+
+  enum MessageType : uint8_t {
+    Tracing = 0,
+    Logging = 1,
+  };
+
+  /* @event */
+  struct EXTERNAL INotification : virtual public Core::IUnknown {
+    enum { ID = ID_MESSAGE_CONTROL_NOTIFICATION };
+    virtual ~INotification() = default;
+    virtual void ReceiveRawMessage(MessageType type, const string &category,
+                                   const string &module, const string &fileName,
+                                   uint16_t lineNumber, uint64_t timestamp,
+                                   const string &message) = 0;
+  };
+
+  virtual void
+  RegisterOutputNotification(IMessageControl::INotification *notification) = 0;
+  virtual void UnregisterOutputNotification(
+      const IMessageControl::INotification *notification) = 0;
+
+  virtual uint32_t Configure(bool isBackground, bool abbreviate,
```

Comment:
Let's be consistent, all parameters we are using as in parameters and thus will not change, lets make the const (I know it is only readability, it has no technical use :-) ) (also check the Notification above :-)